pkgs/applications/audio/radiotray-ng/default.nix

```diff
@@ -89,6 +91,8 @@ stdenv.mkDerivation rec {
   preFixup = ''
     gappsWrapperArgs+=(--suffix PATH : ${lib.makeBinPath [ dbus ]})
     wrapProgram $out/bin/rt2rtng --prefix PYTHONPATH : $PYTHONPATH
+    # for GStreamer
```

Comment:
We really can't use a less invasive variable?

Reply:
not if it needs to work.